Makefile fixes

From: Michal Schmidt <mschmidt@redhat.com>

Fix the 'install' target:
 - Don't assume SBINDIR already exists under DESTDIR.
 - Install the manpage.

Index: latencytop-0.5/Makefile
===================================================================
--- latencytop-0.5.orig/Makefile
+++ latencytop-0.5/Makefile
@@ -1,10 +1,13 @@
 # FIXME: Use autoconf ?
 HAS_GTK_GUI = 1
 
+CC = gcc
+
 DESTDIR =
 SBINDIR = /usr/sbin
+MANDIR  = /usr/share/man
 XCFLAGS = -W  -g `pkg-config --cflags glib-2.0` -D_FORTIFY_SOURCE=2 -Wno-sign-compare
-LDF = -Wl,--as-needed `pkg-config --libs glib-2.0`   -lncursesw 
+LDF = $(LDFLAGS) `pkg-config --libs glib-2.0` -lncursesw
 
 OBJS= latencytop.o text_display.o translate.o fsync.o
 
@@ -26,16 +29,19 @@ endif
 
 # We write explicity this "implicit rule"
 %.o : %.c
-	gcc -c $(CFLAGS) $(XCFLAGS) $< -o $@
+	$(CC) -c $(CFLAGS) $(XCFLAGS) $< -o $@
 
 latencytop:  $(OBJS) latencytop.h Makefile
-	gcc $(CFLAGS) $(OBJS) $(LDF) -o latencytop 
+	$(CC) $(CFLAGS) $(OBJS) $(LDF) -o latencytop
 
 clean:
 	rm -f *~ latencytop DEADJOE *.o
 
 install: latencytop
 	mkdir -p $(DESTDIR)/usr/share/latencytop
+	mkdir -p $(DESTDIR)$(MANDIR)/man8
+	mkdir -p $(DESTDIR)$(SBINDIR)
 	install -m 0644 latencytop.trans $(DESTDIR)/usr/share/latencytop/latencytop.trans
 	install -m 0644 *.png $(DESTDIR)/usr/share/latencytop/
 	install -m 0755 latencytop $(DESTDIR)$(SBINDIR)/
+	install -m 0644 latencytop.8 $(DESTDIR)$(MANDIR)/man8/
